person_pose:
  - -1
  - 1.2
  - 0